restli-common/src/main/pegasus/com/linkedin/restli/common/ExtensionSchemaAnnotation.pdsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "ExtensionSchemaAnnotation",
+  "namespace": "com.linkedin.restli.common",
+  "doc": "Specifies the extension schema annotation configuration for defining the entity relationship among entities.",
+  "fields": [
+    {
+      "name": "resource",
+      "type": "string",
+      "doc": "Specifies resource name to avoid the ambiguity in multi-version scenario."
+    },
+    {
+      "name": "using",
+      "type": "string",
+      "doc": "Specified one to many relationship. It can use either GET_ALL(\"get_all\") or a FINDER(\"finder:<method name>\").",
+      "optional": true
+    },
+    {
+      "name": "params",
+      "type": {
+        "type": "map",
+        "values": "string"
+      },
+      "doc": "Specifies how the injected entity can be fetched using the extendedEntity's fields.",
```

Comment:
This only specifies how parameters if any are specified (or for finders). The actual specification to fetch the entity comes from resource key for GET.

Reply:
Updated the doc.